sse/src/main/scala/io/finch/sse/ServerSentEvent.scala

```diff
@@ -0,0 +1,51 @@
+package io.finch.sse
+
+import java.nio.charset.Charset
+
+import cats.Show
+import com.twitter.concurrent.AsyncStream
+import com.twitter.io.Buf
+import io.finch.{Encode, Ok, Output, Text}
+import io.finch.internal.{BufText, LowPriorityToResponseInstances}
+
+case class ServerSentEvent[A](
+  data: A,
+  id: Option[String] = None,
+  event: Option[String] = None,
+  retry: Option[Long] = None
+)
+
+object ServerSentEvent extends LowPriorityToResponseInstances {
```

Comment:
This is because `asyncResponseBuilder` is inaccessible otherwise. Thinking more about it, I can probably add a `io.finch.sse.internal` package with a `SSEToResponse` that extends this and provides `asyncResponseBuilder`.